test/e2e/common/utils.py

```diff
@@ -232,3 +231,19 @@ def predict_grpc(service_name, payload, version=constants.KSERVE_V1BETA1_VERSION
         options=(('grpc.ssl_target_name_override', host),))
     stub = grpc_predict_v2_pb2_grpc.GRPCInferenceServiceStub(channel)
     return stub.ModelInfer(pb.ModelInferRequest(model_name=model_name, inputs=payload))
+
+
+def predict_modelmesh(service_name, input_json, model_name=None):
+    with open(input_json) as json_file:
+        data = json.load(json_file)
+        cluster_ip = get_cluster_ip("modelmesh-serving", "default")
```

Comment:
this is internal service which does not have cluster ip, we might need to do port forward here.